iceoryx_posh/include/iceoryx_posh/popo/wait_set.hpp

```diff
@@ -22,6 +22,8 @@
 #include "iceoryx_posh/internal/popo/building_blocks/condition_variable_data.hpp"
 #include "iceoryx_posh/popo/enum_trigger_type.hpp"
 #include "iceoryx_posh/popo/event_attorney.hpp"
+#include "iceoryx_posh/popo/event_callback.hpp"
+#include "iceoryx_posh/popo/event_info.hpp"
```

Comment:
This include can be removed.

Reply:
Nope, it is used in `using EventInfoVector = cxx::vector<const EventInfo*, CAPACITY>;` line 66 and the implementation

Reply:
`iceoryx_posh/popo/trigger.hpp` already includes `event_info.hpp` as the class `Trigger` has a member EventInfo m_eventInfo`. So I think in case of implicit dependencies we don't need the additional include?